Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -440,4 +454,16 @@ def check_for_duplicate_pull_requests(formula)
       EOS
     end
   end
+
+  def alias_update_pair(formula, new_formula_version)
+    versioned_alias = formula.aliases.grep(/.@\d/).first
+    return if versioned_alias.nil?
+
+    name, old_alias_version = versioned_alias.split("@")
+    version_regex = (old_alias_version.split(".").length == 1) ? /^\d+/ : /^\d+\.\d+/
```

Comment:
Making this an `if`/`else` would make it a little quicker to mentally parse.

Reply:
I hope you had something like this in mind? (If we are using a whole `if...else` statement than we can just inline the regex.)
https://github.com/Homebrew/brew/blob/e605713be8837a33407a23e82be160913489f5b9/Library/Homebrew/dev-cmd/bump-formula-pr.rb#L437-L443